# Intel compiler have problems with precompiled headers (cannot mix C and C++ precompiled headers, like in GNU compilers)
# B. Hajgato Nov 07 2014
--- wxPython-src-3.0.1.1.ORIG/build/tools/build-wxwidgets.py	2014-01-04 22:08:37.000000000 +0100
+++ wxPython-src-3.0.1.1/build/tools/build-wxwidgets.py	2014-11-07 14:02:02.144031589 +0100
@@ -265,6 +265,7 @@
             configure_opts.append("--with-osx_cocoa")
 
         wxpy_configure_opts = [
+                            "--disable-precomp-headers",
                             "--with-opengl",
                             "--enable-sound",
                             "--enable-graphics_ctx",
